UI/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Politico HomePage</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="./styles/index.css" />
+</head>
+<body>
+    <header>
+        <header>
+            <nav>
+                <div class="navigation__wrapper">
+                    <div class='container'>
+                    <div><a href="#" id="header-title">Politico</a></div>
+                    <div class='nav-link'><a href="./views/admin/admin.html">Admin</a></div>
+                    <div class='nav-link'><a href="./politician.html">Candidate</a></div>
+                    <div class='nav-link'><a href="./views/user/user-dashboard.html">My Profile</a></div>
+                    <div class='nav-link'><a href="./login.html">Log In</a></div>
+                    </div>
+                </div>
+            </nav>
+    </header>
+
+    <section id='welcome'>
+        <div class='container'>
+            <div id='welcome-box'>
+                <div class='welcome-text'>
+                    <h1>decide your <span>future</span> now!</h1>
+                    <button id="signup-btn"><a id='btn' href="views/resgister.html">Sign Up</a></button>
+                </div>
```

Comment:
Could you correct the spelling of your sign up url

Reply:
Done, changes would reflect in the next PR request... 
Thank you,